components/server/src/workspace/workspace-starter.ts

```diff
@@ -1852,8 +1854,8 @@ export class WorkspaceStarter {
             targetMode = CloneTargetMode.REMOTE_HEAD;
         }
 
-        const tokenValidityThreshold = (await isLongAccessTokenValidityThresholdEnabled(user)) ? 50 : 30;
```

Comment:
The default for "start workspace" was 30 minutes before. We are reducing to 10 [here](https://github.com/gitpod-io/gitpod/pull/19715/files#diff-b1c591e18af598ccba93b570ce7c97dece0988c9109a6d3be82bbe0df1fbd619R147), but can adjust any time with the new [feature flag](https://github.com/gitpod-io/gitpod/pull/19715/files#diff-b1c591e18af598ccba93b570ce7c97dece0988c9109a6d3be82bbe0df1fbd619R1986).